support/package_modsecurity

```diff
@@ -0,0 +1,102 @@
+ar x libyajl-dev_${yajl_version}_amd64.deb
+tar --extract --xz --verbose --directory "$install_dir" --file data.tar.xz
+ar x libyajl2_${yajl_version}_amd64.deb
+tar --extract --xz --verbose --directory "$install_dir" --file data.tar.xz
+
+echo "-----> Packaging liblmdb"
+lmdb_version="0.9.24-1"
+curl --remote-name --location "http://mirrors.kernel.org/ubuntu/pool/main/l/lmdb/liblmdb-dev_${lmdb_version}_amd64.deb"
+curl --remote-name --location "http://mirrors.kernel.org/ubuntu/pool/main/l/lmdb/liblmdb0_${lmdb_version}_amd64.deb"
+ar x liblmdb-dev_${lmdb_version}_amd64.deb
+tar --extract --xz --verbose --directory "$install_dir" --file data.tar.xz
+ar x liblmdb0_${lmdb_version}_amd64.deb
+tar --extract --xz --verbose --directory "$install_dir" --file data.tar.xz
+
+echo "-----> Packaging ssdeep (libfuzzy)"
+libfuzzy_version="2.14.1+git20180629.57fcfff-1build1"
+curl --remote-name --location "http://mirrors.kernel.org/ubuntu/pool/universe/s/ssdeep/libfuzzy2_${libfuzzy_version}_amd64.deb"
+curl --remote-name --location "http://mirrors.kernel.org/ubuntu/pool/universe/s/ssdeep/libfuzzy-dev_${libfuzzy_version}_amd64.deb"
+ar x libfuzzy2_${libfuzzy_version}_amd64.deb
+tar --extract --xz --verbose --directory "$install_dir" --file data.tar.xz
+ar x libfuzzy-dev_${libfuzzy_version}_amd64.deb
+tar --extract --xz --verbose --directory "$install_dir" --file data.tar.xz
+
+echo "-----> Packaging Lua (liblua5.1)"
+lua_version="5.3.3-1.1ubuntu2"
+curl --remote-name --location "http://mirrors.kernel.org/ubuntu/pool/main/l/lua5.3/liblua5.3-0_${lua_version}_amd64.deb"
+curl --remote-name --location "http://mirrors.kernel.org/ubuntu/pool/main/l/lua5.3/liblua5.3-dev_${lua_version}_amd64.deb"
+ar x liblua5.3-0_${lua_version}_amd64.deb
+tar --extract --xz --verbose --directory "$install_dir" --file data.tar.xz
+ar x liblua5.3-dev_${lua_version}_amd64.deb
+tar --extract --xz --verbose --directory "$install_dir" --file data.tar.xz
+
+echo "-----> Downloading modsecurity"
+curl --remote-name --location "$MODSECURITY_URL"
+tar --extract --ungzip --verbose --file "$MODSECURITY_TARBALL"
+pushd "$MODSECURITY_DIR" >/dev/null
+  ./build.sh && \
+  # SSDEEP is not handled like others, it's not possible to easily tweak the path, doing it manually
+  sed -i "s+^SSDEEP_POSSIBLE_PATHS=.*+SSDEEP_POSSIBLE_PATHS=${install_dir}/usr+" "./configure" && \
+  sed -i "s+^LUA_POSSIBLE_PATHS=.*+LUA_POSSIBLE_PATHS=${install_dir}/usr+" "./configure" && \
+  sed -i "s+LUA_CFLAGS=\"-I\${lua_inc_path}\"+LUA_CFLAGS=\"-I\${lua_inc_path} -I\${lua_inc_path}/../x86_64-linux-gnu\"+" "./configure" && \
+  ./configure \
+     "--prefix=${install_dir}" \
+     "--with-geoip=${install_dir}/usr" \
+     "--with-yajl=${install_dir}/usr" \
+     "--with-lmdb=${install_dir}/usr" \
+     "--with-ssdeep=yes" \
+     "--with-lua=${install_dir}/usr" \
+     --disable-examples && \
+  make && \
+  make install
```

Comment:
Do we really need the `make install` instruction?
```suggestion
  make
```

Reply:
Yes we do, since we've defined the installation prefix, we want to package what is installed